src/filters/string.rs

```diff
@@ -93,6 +94,69 @@ pub fn addslashes(value: Value, _: HashMap<String, Value>) -> TeraResult<Value>
+    let split_pattern = Regex::new(r"([-\s\(\{\[<]+)").unwrap();
+    let words = split_pattern.split(&s).collect::<Vec<&str>>();
+    let capitalized = words.iter()
+                           .map(|&word| {
+                               let mut characters = word.chars();
+                               match characters.next() {
+                                   None => "".to_string(),
+                                   Some(f) => {
+                                       f.to_uppercase().collect::<String>() +
+                                       &characters.as_str().to_lowercase()
+                                   }
+                               }
+                           }).filter(|word|{
+                           ! word.is_empty()
+                           })
+                           .collect::<Vec<_>>();
+
+    match split_pattern.find(&s) {
+        None => Ok(to_value(&capitalized.join(""))),
+        Some(first_index) => {
+            let captures = split_pattern.captures_iter(&s);
+            let mut result: String;
+            if first_index.0 == 0 {
+                result = captures.zip(capitalized.iter())
+                                 .map(|(seperator, word)| {
+                                     let mut temp = seperator.at(0).unwrap().to_string();
+                                     temp.push_str(word);
+                                     temp
+                                 })
+                                 .collect::<Vec<String>>()
+                                 .join("");
+            } else {
+                result = capitalized.iter()
+                                    .zip(captures)
+                                    .map(|(word, seperator)| {
+                                        let mut temp = word.clone();
+                                        temp.push_str(seperator.at(0).unwrap());
+                                        temp.to_string()
+                                    })
+                                    .collect::<Vec<String>>()
+                                    .join("");
+            }
+            let captured_patterns_count = split_pattern.find_iter(&s).count();
+            if captured_patterns_count > capitalized.len() {
+                result.push_str( split_pattern.captures_iter(&s).last().unwrap().at(0).unwrap());
+            } else if capitalized.len() > captured_patterns_count {
+                result.push_str(capitalized.last().unwrap());
+            }
+            Ok(to_value(&result))
+        }
+    }
+}
+
+///Removes html tags from string
+pub fn striptags(value : Value, _: HashMap<String, Value>) -> TeraResult<Value> {
+    let s = try_get_value!("striptags", "value", String, value);
+    let tag_pattern = Regex::new(r"(<!--.*?-->|<[^>]*>)").unwrap();
+    Ok(to_value(&tag_pattern.split(&s).map(|x| x.replace("\\n","").trim().to_string()).filter(|x| !x.is_empty()).collect::<Vec<String>>().join(" ")))
```

Comment:
using django 1.10:

```
In [5]: strip_tags("<p> blalb \n erer</p>")
Out[5]: ' blalb \n erer'
```
That might just be one of their test quirk, you can just remove that special handling